lib/src/main/java/com/diffplug/spotless/npm/NpmProcess.java

```diff
@@ -38,7 +38,8 @@ void install() {
 	}
 
 	Process start() {
-		return npm("start");
+		// adding --scripts-prepend-node-path=true due to https://github.com/diffplug/spotless/issues/619#issuecomment-648018679
+		return npm("start", "--scripts-prepend-node-path=true");
```

Comment:
As a reaction to the discussion in #619 - make sure we always use the node binary associated with the npm binary we use.